hack/jenkins/e2e-runner.sh

```diff
@@ -21,96 +21,124 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
+# TODO(ihmccreery) I'm not sure if this is necesssary, with the workspace check
+# below.
+function clean_binaries() {
+    echo "Cleaning up binaries."
+    rm -rf kubernetes*
+}
+
+function fetch_server_version_tars() {
+    msg=$(gcloud ${CMD_GROUP} container get-server-config --project=${PROJECT} --zone=${ZONE} | grep defaultClusterVersion)
+    # msg will look like "defaultClusterVersion: 1.0.1". Strip
+    # everything up to, including ": "
+    build_version="v${msg##*: }"
+    fetch_tars "release" "${build_version}"
+}
+
+# use JENKINS_PUBLISHED_VERSION, for CI
+#
+# Use a published version like "ci/latest" (default), "release/latest",
+# "release/latest-1", or "release/stable"
+function fetch_published_version_tars() {
+    JENKINS_PUBLISHED_VERSION=${JENKINS_PUBLISHED_VERSION:-'ci/latest'}
```

Comment:
It would be real spiffy if this were an argument to the function instead of an environment variable.